Update playbooks_conditionals.rst (#249)
ansible/ansible-documentation: docs/docsite/rst/playbook_guide/playbooks_conditionals.rst

      - ansible.builtin.shell: echo "only on Red Hat 6, derivatives, and later"
        when: ansible_facts['os_family'] == "RedHat" and ansible_facts['lsb']['major_release'] | int >= 6

> Storing an ansible fact using variable and then using this varaible for a conditional logic 

Reviewer: ```suggestion
You can storing Ansible facts as variables to use for conditional logic, as in the following example:
```

      - ansible.builtin.shell: echo "only on Red Hat 6, derivatives, and later"
        when: ansible_facts['os_family'] == "RedHat" and ansible_facts['lsb']['major_release'] | int >= 6

> You can storing Ansible facts as variables to use for conditional logic, as in the following example:

Reviewer: ```suggestion
You can store Ansible facts as variables to use for conditional logic, as in the following example:
```
